do_cmake.sh

```diff
@@ -1,7 +1,9 @@
 #!/usr/bin/env bash
 set -ex
 
-git submodule update --init --recursive
+if [ -d .git ]; then
```

Comment:
could you s/build/do_cmake.sh/ in the title of this commit? more specific this way.

Reply:
Squashed as requested